examples/read_workplaces_with_industry_naics.py

```diff
@@ -1,10 +1,7 @@
-import numpy as np
-import sciris as sc
 import synthpops as sp
 
-
 if __name__ == '__main__':
-    n = int(50e3)
+    n = int(5e4)
```

Comment:
Given the convention for thinking of populations in thousands, would suggest reverting this change (and elsewhere, Xe4 should probably be better as X0e3)